tutorials/audio/audio_streams.rst

```diff
@@ -28,15 +28,23 @@ played back often.
 AudioStreamPlayer
 -----------------
 
-.. image:: img/audio_stream_player.png
+.. image:: img/audio_stream_player.webp
 
 This is the standard, non-positional stream player. It can play to any bus.
 In 5.1 sound setups, it can send audio to stereo mix or front speakers.
 
+Playback Type is an experimental setting, and could change in future versions
+of Godot. It exists so web exports use samples for audio instead of streams,
+this prevents the audio from being garbled in single threaded web exports. By
+default only the web platform will use samples, changing this is not recommended
+unless you have an explicit reason to. You can change the default playback type
+for the web and other platforms in the project settings under **Audio > General**
+(advanced settings must be turned on to see the setting).
```

Comment:
```suggestion
Playback Type is an experimental setting, and could change in future versions
of Godot. It exists so web exports use Web Audio based samples instead of streaming all sounds to the browser, unlike most platforms.
This prevents the audio from being garbled in single-threaded web exports. By
default, only the Web platform will use samples. Changing this setting is not recommended, unless you have an explicit reason to. You can change the default playback type
for the web and other platforms in the project settings under **Audio > General**
(advanced settings must be turned on to see the setting).
```